Src/BootCamp.Chapter/LogDemo.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    internal class LogDemo
+    {
+        internal static void Demo(Illogger logger)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            logger.Log($"INFO  {DateTime.Now} The programm has started {Environment.NewLine}");
+            Console.ForegroundColor = ConsoleColor.White;
+
+            for (int i = 0; i < 2; i++)
+            {
+                Console.Write("What is your name: ");
```

Comment:
Let's not forget what we learned before. Please move the body of for loop into a function which tellls what it does.